src/med_bench/mediation.py

```diff
@@ -323,7 +323,7 @@ def mediation_coefficient_products(y, t, m, x, interaction=False, regularization
             None]
 
 
-def mediation_g_formula(y, t, m, x, interaction=False, forest=False,
+def mediation_g_computation(y, t, m, x, interaction=False, forest=False,
```

Comment:
please revert to mediation_g_formula, the g_estimator (in R) and this are not the same estimator. mediation_g_formula is not the python version of the g_estimator. g_estimator is the name given in the R package, and I am not sure we should change it. For the python version, I am not sore about the official name between g_formula or g_computation